spec/02-integration/22-ai_plugins/01-reports_spec.lua

```diff
@@ -0,0 +1,237 @@
+local helpers = require "spec.helpers"
+local constants = require "kong.constants"
+local cjson = require "cjson"
+local pl_file = require "pl.file"
+
+local PLUGIN_NAME = "ai-proxy"
+local MOCK_PORT = helpers.get_available_port()
+
+for _, strategy in helpers.each_strategy() do
+  local admin_client
+  local dns_hostsfile
+  local reports_server
+
+  describe("anonymous reports for ai plugins #" .. strategy, function()
+    local reports_send_ping = function(port)
+      ngx.sleep(0.2) -- hand over the CPU so other threads can do work (processing the sent data)
```

Comment:
Use the `eventually` helper to avoid bare sleeps and flakyness

Reply:
Done. Thanks